google-cloud-speech/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+### 0.32.1 / 2019-01-22
+
+- Re-generate google-cloud-speech files (#2819)
```

Comment:
This is the wrong level of information to put in the changelog. The changes should be listed, not the way the changes were made.

```
* Add the following attributes to the Google::Cloud::Speech::V1 namespace:
  * RecognitionConfig#enable_separate_recognition_per_channel
  * StreamingRecognitionResult#channel_tag
  * SpeechRecognitionResult#channel_tag
```